libs/SalesforceSDK/src/com/salesforce/androidsdk/ui/ServerPickerActivity.java

```diff
@@ -221,7 +227,10 @@ public CustomServerUrlEditor getCustomServerUrlEditor() {
      * Refreshes the authorization configuration.
      */
     private void reconfigureAuthorization() {
-        (new AuthConfigTask(this)).execute();
+        SalesforceSDKManager.getInstance().fetchAuthenticationConfiguration(() -> {
```

Comment:
Here's a call site for one of the new Kotlin Coroutine jobs from Java, which looks pretty nice!  The rest of this diff is trivial inspector cleanup.